posthog/api/survey.py

```diff
@@ -878,19 +898,28 @@ def _calculate_rates(self, stats: SurveyStats) -> SurveyRates:
         rates: SurveyRates = {
             "response_rate": 0.0,
             "dismissal_rate": 0.0,
+            "unique_users_response_rate": 0.0,
+            "unique_users_dismissal_rate": 0.0,
         }
 
         shown_count = stats["survey shown"]["total_count"]
         if shown_count > 0:
             sent_count = stats["survey sent"]["total_count"]
             dismissed_count = stats["survey dismissed"]["total_count"]
+            unique_users_shown_count = stats["survey shown"]["unique_persons"]
```

Comment:
lets move those keys to constants, we are repeating all over the place
no reason to DRY, and not apply some https://en.wikipedia.org/wiki/Defensive_programming

Reply:
i totally agree, but in this case, they are already enforced by our [type system here](https://github.com/PostHog/posthog/blob/feat/improvements-for-survey-stats/posthog/api/survey.py#L58-L86). so we already get intelisense on those names, and if we choose another name, the linter will catch them